src/components/common/copyLinkButton/CopyLinkButton.tsx

```diff
@@ -31,6 +31,13 @@ export const CopyLinkButton = ({
       }
     });
 
+    displayNotification(
```

Comment:
Do you think this should live in the `.then`? It probably won't make a difference, but in case, for whatever reason the `clipboard.writeText` fails, if we don't have it in the `then` block we'd show a success message